tests/test_modeling_tf_common.py

```diff
@@ -1042,6 +1042,7 @@ def test_model_common_attributes(self):
 
         for model_class in self.all_model_classes:
             model = model_class(config)
+            model(model.dummy_inputs)  # build the model
```

Comment:
needed for the `get_bias()` call below --` model.bias_layer.bias` doesn't exist unless the model gets built

Reply:
`needed for the get_bias() call below ` Would be a better comment that `build the model` :)